examples/orgpolicy/v1beta1/policy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: orgpolicy.gcp.upbound.io/v1beta1
+kind: Policy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: orgpolicy/v1beta1/policy
+    upjet.upbound.io/manual-intervention: "Policy requires Org Access Rights"
+    crossplane.io/external-name: iam.disableServiceAccountKeyUpload
```

Comment:
I'm a little confused here. As far as I understand, we read the name from the annotation. Does it match one of the [formats](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/org_policy_policy#name) in the Terraform document? Does the import test work like this?

Reply:
in the terraform documentation you see `{{parent}}/policies/{{name}}` which means you need to set spec.forProvider.parent and the name is the `crossplane.io/external-name`

for example `projects/crossplane-playground/policies/iam.disableServiceAccountKeyUpload` is it better in this case to use `IdentifiedByProvider` ?

Reply:
If the import test passes successfully, I think it would be better to keep it that way in terms of normalization. If not, we can use `config.IdentifierFromProvider`.

Reply:
yes import test is working